framework/decode/vulkan_replay_dump_resources_compute_ray_tracing.cpp

```diff
@@ -900,7 +910,23 @@ VkResult DispatchTraceRaysDumpingContext::DumpDispatchTraceRays(
             }
         }
 
-        GenerateOutputJsonTraceRaysIndex(qs_index, bcb_index, tr_index, i);
+        VulkanDumpDrawCallInfo draw_call_info{};
+        draw_call_info.type                         = DumpResourceType::kTraceRaysIndex;
+        draw_call_info.instance_table               = instance_table;
+        draw_call_info.device_table                 = device_table;
+        draw_call_info.object_info_table            = &object_info_table;
+        draw_call_info.device_info                  = device_info;
+        draw_call_info.original_command_buffer_info = original_command_buffer_info;
+        draw_call_info.bcb_index                    = bcb_index;
+        draw_call_info.cmd_buf_index                = i;
```

Comment:
Here you need `draw_call_info.cmd_index = tr_index;` instead of `draw_call_info.cmd_buf_index = i;`

Reply:
Same for this for loop convert it into
```
    for (const uint64_t tr_index : trace_rays_indices)
    {
```